File: planet-id-demo-rp-web/src/main/java/com/planetway/fudosan/domain/Person.java
package com.planetway.fudosan.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Person {
    private String firstNameKatakana;
    private String lastNameKatakana;
    private String firstNameKanji;
    private String lastNameKanji;
    private String firstNameRomaji;
    private String lastNameRomaji;
    private String dateOfBirth;
    private List<Address> address;
}

File: planet-id-demo-rp-android/app/src/main/java/com/planetway/demo/fudosan/data/model/Person.kt
package com.planetway.demo.fudosan.data.model

data class Person(
    var firstNameKatakana: String?,
    var lastNameKatakana: String?,
    var firstNameKanji: String?,
    var lastNameKanji: String?,
    var firstNameRomaji: String?,
    var lastNameRomaji: String?,
    var dateOfBirth: String?,
    var address: List<Address>?
)
